lib/adapter.js

```diff
@@ -740,7 +739,7 @@ function Adapter(options) {
                     const chained = readFileCertificate(obj.native.certificates[chainedName]).split('-----END CERTIFICATE-----\r\n');
                     ca = [];
                     for (let c = 0; c < chained.length; c++) {
-                        if (chained[c].replace(/[\r\n|\r|\n]+/, '').trim()) {
+                        if (chained[c].replace(/(\r\n|\r|\n)/g, '').trim()) {

```

Comment:
I'm not sure here - are we replacing multiple line breaks with `''` or all of them?
Anyways, the old code was replacing `\r`, `\n` and `|`.

Reply:
good question ... @GermanBluefox 
I would say change is ok, but also have no 100% overview

Reply:
I expect we want to kill all linebreaks here

Reply:
It is ok.